site/content/faq/what-is-the-default-directory-that-zap-uses.md

```diff
@@ -7,35 +7,23 @@ weight: 2
 
 The default directory that ZAP uses depends on the OS.
 
-Note that you can override this using the -dir [command line](/docs/desktop/cmdline/) option.
+It can be overridden using the `-dir` [command line](/docs/desktop/cmdline/) option.
 
-##  Windows 7 / 8
+The default or home directory contains ZAP files added or modified at runtime. Including (but not limited to): config files, add-ons, input files, logs, etc. It is also the save location, for: scripts, active scan policy files, exported contexts, etc.
```

Comment:
It is also the _default_ save location ... - users can change that

Reply:
But not for everything mentioned.

Reply:
Per comments yesterday I thought the default save location was user home (not zap home).

Reply:
If someone else wants to suggest different text I'm happy to update the PR.